packages/react/src/themes/base/components/Checkbox/checkboxVariables.ts

```diff
@@ -0,0 +1,77 @@
+import { pxToRem } from '../../../../lib'
+
+export type CheckboxVariables = {
+  checkboxBackground: string
+  checkboxBorderColor: string
+  checkboxBorderStyle: string
+  checkboxBorderRadius: string
+  checkboxBorderWidth: string
+  checkboxColor: string
+  checkboxMargin: string
+  checkboxPadding: string
+
+  toggleBackground: string
+  toggleBorderColor: string
+  toggleBorderStyle: string
+  toggleBorderRadius: string
+  toggleBorderWidth: string
+  toggleColor: string
+  toggleMargin: string
+  togglePadding: string
+
+  checkedCheckboxBackground: string
+  checkedCheckboxBorderColor: string
+  checkedCheckboxColor: string
+  checkedToggleColor: string
+  checkedToggleBackground: string
+  checkedToggleBorderColor: string
+  checkedTogglePadding: string
+
+  disabledColor: string
+  disabledCheckboxBackground: string
+  disabledCheckboxBorderColor: string
+  disabledCheckboxColor: string
+  disabledToggleColor: string
+  disabledToggleBackground: string
+  disabledToggleBorderColor: string
+
+  padding: string
+}
+
+export default (siteVars: any): CheckboxVariables => ({
+  checkboxBackground: 'transparent',
+  checkboxBorderColor: siteVars.colors.grey[300],
+  checkboxBorderStyle: `solid`,
+  checkboxBorderRadius: pxToRem(4),
+  checkboxBorderWidth: pxToRem(1),
+  checkboxColor: 'transparent',
+  checkboxMargin: `0 ${pxToRem(12)} 0 0`,
+  checkboxPadding: '0',
+
+  toggleBackground: 'transparent',
+  toggleBorderColor: siteVars.colors.grey[300],
+  toggleBorderStyle: `solid`,
+  toggleBorderRadius: pxToRem(8),
```

Comment:
nit: can we make this value bigger, it looks better that way:
before pxToRem(8)
![image](https://user-images.githubusercontent.com/4512430/58795400-d9913080-85fa-11e9-81d1-1f30080fcf5d.png)

after pxToRem(999)
![image](https://user-images.githubusercontent.com/4512430/58795354-b9fa0800-85fa-11e9-9ba3-c32f3e24e2fb.png)